### ISSUE TITLE
 'NoneType' object has no attribute 'split'

### ISSUE THREAD

**Issue by pilate** [fireeye/flare-floss#209]
61ebd59d411d97fdc7cff67d620c0add

```
ERROR:floss:Vivisect failed to load the input file: 'NoneType' object has no attribute 'split'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/floss-1.2.0-py2.7.egg/floss/main.py", line 600, in main
    vw = viv_utils.getWorkspace(sample_file_path, should_save=options.save_workspace)
  File "/usr/local/lib/python2.7/dist-packages/viv_utils-0.3.5-py2.7.egg/viv_utils/__init__.py", line 32, in getWorkspace
    vw.loadFromFile(fp)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/vivisect/__init__.py", line 2134, in loadFromFile
    fname = mod.parseFile(self, filename)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/vivisect/parsers/pe.py", line 30, in parseFile
    return loadPeIntoWorkspace(vw, pe, filename)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/vivisect/parsers/pe.py", line 96, in loadPeIntoWorkspace
    dllname = pe.getDllName()
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/PE/__init__.py", line 410, in getDllName
    return rawname.split('\x00')[0]
AttributeError: 'NoneType' object has no attribute 'split'
```

```
$ ./floss 59fd11e1-7a5b-5e97-8047-7db7c81011d5
ERROR:floss:Vivisect failed to load the input file: 'NoneType' object has no attribute 'split'
```

**Comment by williballenthin** [comment]
vivisect issue, need to triage and report upstream

**Comment by mr-tz** [comment]
still present in current vivisect

**Comment by mr-tz** [comment]
fixed in vivisect 1.0.4
